<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MostrarVehiculoUi</class>
 <widget class="QWidget" name="MostrarVehiculoUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelNombre">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre vehiculo</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelColor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Color:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NombreText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>311</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelRuedas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Número de ruedas: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMotor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Potencia motor:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCombustible">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo combustible:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMatricula">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>550</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Num matrícula:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelReparacion">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reparación:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMotor_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Motor: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCombustible_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Combustible: </string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAlas">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Alas:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelReactores">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Reactores:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>640</y>
     <width>311</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelNumVagones">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>470</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Numero de Vagones:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLocomotora">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Locomotora:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumRuedasText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="MotorText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="PotenciaText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="CombText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="AlasText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ReactoresText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="TipoCombText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ColorText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LocomotoraText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="NumVagonesText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>470</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ReparacionText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>510</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="MatriculaText">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>550</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelTrenAterrizaje">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>111</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Tren aterrizaje:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelCategoria">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>590</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Categoria:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCategoria">
   <property name="geometry">
    <rect>
     <x>182</x>
     <y>590</y>
     <width>171</width>
     <height>25</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
